src/foam/nanos/auth/GlobalReadAuthorizer.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.nanos.auth;
+
+import foam.core.FObject;
+import foam.core.X;
+
+public class GlobalReadAuthorizer extends StandardAuthorizer {
+
+  // this authorizer is an extension of the standardauthorizer that is meant
+  // to replace the authorizeReads flag on services using the standardAuthorizer.
+  // setAuthorize(true) => setAuthorizer(new foam.nanos.auth.GlobalReadAuthorizer(\"permissionPrefix\"))
+
+  public GlobalReadAuthorizer(String permissionPrefix) {
+    super(permissionPrefix);
+  }
+
+  @Override 
+  public void authorizeOnRead(X x, FObject obj) throws AuthorizationException {
```

Comment:
I think it's ok if class inherits this method from super since it's only called when checkGlobal returns false

Reply:
makes sense will remove and add globalread check for find back to authorizationdao